src/app/components/Headline/index.jsx

```diff
@@ -10,7 +11,11 @@ const StyledHeadline = styled.h1`
 `;
 
 const Headline = ({ blocks }) => {
-  const { text } = blocks[0].model.blocks[0].model;
+  const { text } = extractText(blocks);
+
+  if(!text){
```

Comment:
Surprised linting isn't complaining about this. A trivial change: add spacing to this conditional, e.g. `if (!text) {`